pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -84,6 +84,13 @@ type TLSSpec struct {
 	IssuerConf *cmmeta.ObjectReference `json:"issuerConf,omitempty"`
 }
 
+const (
+	UpgradeStrategyDisabled = "disabled"
+	UpgradeStrategyNever    = "never"
+)
+
+const DefaultVersionServiceEndpoint = "https://check.percona.com"
```

Comment:
We need a way to override default endpoint, please see: https://github.com/percona/percona-server-mongodb-operator/blob/main/pkg/apis/psmdb/v1/psmdb_types.go#L201-L207

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1236/commits/479e6792a0c6864e5c8711dadbb82fe43730735c